packages/es-components/src/components/patterns/datepicker/DatePicker.js

```diff
@@ -53,7 +37,8 @@ export const DatePicker = props => {
   /* eslint-enable */
 
   const textbox = (
-    <DateTextbox
+    <Textbox
+      inputRef={() => React.createRef()}
```

Comment:
Is that a reference to the Textbox's `inputRef` prop?

Reply:
Correct. That `customInputRef` will tell `react-datepicker` to apply its ref function to `inputRef` instead of `ref,` which is what causes the focus issues.

Reply:
Ah, never mind. I looked at the code. I can make this work.